network/connection_test.go

```diff
@@ -19,20 +19,14 @@ func TestDefaultEndpoint(t *testing.T) {
 	resolver := resolver{}
 	serverIps := []netip.Addr{
 		netip.MustParseAddr("1.1.1.1"),
-		netip.MustParseAddr("2606:4700:4700::1111"),
 	}
 	endpoint := DefaultEndpoint(&resolver, serverIps)
 
 	ip, err := endpoint.ip()
 	assert.NoError(t, err)
-	assert.Equal(t, serverIps[1], ip)
+	assert.Equal(t, serverIps[0], ip)
```

Comment:
Isn't it testing the same thing? `ip()` now just delegates to `ip4()` so this is effectively the same block as below